test/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
+LABEL name="EL7 OSG 3.5 Gratia Probe Test Container"
+
+RUN cd \
+    && yum install -y yum-priorities \
+    && yum install -y \
+       http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+    && yum install -y \
+       http://repo.opensciencegrid.org/osg/3.5/osg-3.5-el7-release-latest.rpm \
```

Comment:
Any reason you're doing this instead of using `software-base`? I can go either way since this way we're not bringing in supervisor

Reply:
I actually looked at `opensciencegrid/software-base:fresh` first, but when i tried to docker-run that, it seemed to be "doing something" rather than just giving me a prompt; so i decided to just make a new one.  Otherwise i'm not against using software-base if that can be made to work.

Reply:
The "doing something" is just supervisor (service management for containers) and we don't have to use it here but it may actually be nice to have it just "hang". That way your dev workflow would look liike:

1. `docker run -v <path to host condor_meter>:<wherever condor_meter goes in the container>--name my-probe-dev-env --detach gratia-probe`
1. Make changes to `condor_meter` on your host
1. `docker exec -it my-probe-dev-env condor_meter <options>`
1. See output from above, make additional changes to `condor_meter`, etc.

Reply:
Is there a way to log in and look around at files (logs, quarantine) ?

Or do you have to mount those dirs also?

Reply:
Yup: `docker exec -it my-probe-dev-env /bin/sh`